File: spb-admin/api/schemas/ErrorLog.js
const mongoose = require("mongoose");

const { Schema } = mongoose;
const ErrorLogSchema = new Schema(
  {
    _id: Object,
    user_id: String,
    user_ip: String,
    status: Number,
    message: String,
    exception: String,
    timestamp: Date,
    _class: String,
  },
  { collection: "error-log" }
);

module.exports = mongoose.model("error-log", ErrorLogSchema);

File: spb/src/main/java/spb/ubooks/mongoEntity/ErrorLog.java
package spb.ubooks.mongoEntity;

import java.time.LocalDateTime;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import lombok.Data;
import lombok.ToString;

@Data
@Document("error-log")
@ToString
public class ErrorLog {
	
	@Id
	private String _id;
	private String user_id = null;
	private String user_ip;
	private int status;
	private String url;
	private String message;
	private String exception;
	private LocalDateTime timestamp = LocalDateTime.now();
}
